pixart.js

```diff
@@ -1 +1,43 @@
+var brush = document.querySelector(".brush");
+var button = document.querySelector("#set-color");
+var colorInput = document.querySelector("#color-field");
+var color = "#1B4370";
 
+function startGame() {
+  button.addEventListener("click", setColorButton);
```

Comment:
Nice job! I love that this function is just in charge of setting up event listeners, great way to make your code more modular :+1: